### ISSUE TITLE
Sidebar not visible in TinyMCE 5

### ISSUE THREAD

**Issue by pl12133** [tinymce/tinymce#4936]
**Do you want to request a *feature* or report a *bug*?**

Report a bug

**What is the current behavior?**

Sidebar is not visible in TinyMCE version 5.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [fiddle.tinymce.com](http://fiddle.tinymce.com/) or similar.**

`tinymce@5.0.3`: http://fiddle.tinymce.com/LFgaab/3
`tinymce@4.9.4`: http://fiddle.tinymce.com/MFgaab

**What is the expected behavior?**

Sidebar should be visible in the example.

**Which versions of TinyMCE, and which browser / OS are affected by this issue? Did this work in previous versions of TinyMCE?**

Affected version: `>=5.0.0`
Working versions: `<=4.9.4`

----

Am I missing something here? I've read all the docs I could find but can't get a Sidebar to show up in TinyMCE 5.

**Comment by shivanis22** [comment]
Hi,
I was going through the documentation for version 5 and I found a few changes mentioned in the Release Notes regarding sidebars:

https://www.tiny.cloud/docs/release-notes/release-notes50/#customsidebars

The **onrender** should be **onSetup**. Also, noted the camel case notation being used for the onHide and onShow.

But even with these changes, I could not get the sidebar to show. And then I checked the demos for the latest version and found the sidebar is specified in the "toolbar". 
So I made these changes to your code and it worked:

http://fiddle.tinymce.com/ZFgaab/1

Note that the icon for the sidebar is now displayed in the top toolbar and not on the side. 

![image](https://user-images.githubusercontent.com/25460299/56081257-d8832480-5e3d-11e9-9a7a-aabbdf885c90.png)

I could not find any documentation updates for these changes.

I hope this helps.
